spec/values/calculation/round/two_arguments.hrx

```diff
@@ -134,10 +132,70 @@ a {
 
 <===>
 ================================================================================
-<===> infinity/input.scss
+<===> infinity_and_positive/input.scss
```

Comment:
I don't think you need to be as thorough as this for the two-argument `round()`, since it's effectively already tested by `strategy/nearest`. Just have enough tests to verify that it's using the `nearest` strategy at all.

Reply:
None of these are repeated, here we test with the `number` as infinity, and in `nearest.hrx` we test `step` as infinity. Since all of these edge cases are being tested in [wpt](https://github.com/web-platform-tests/wpt/blob/master/css/css-values/round-function.html), I think we should keep them, what do you think?

Reply:
It's not clear to me why the WPT test some cases with explicit `nearest` and some without. I think for us, it probably makes more sense to move all of these explicitly into `round/strategy/nearest`. A casual reader won't know what's from WPT and what isn't, so that doesn't make for a great organizing principle.